README.md

```diff
@@ -48,56 +48,36 @@ Once your account is approved, contact us with your username and ask to be added
 
 Below is a quick reference guide for standard installation.  For more complete details and troubleshooting, see the [Full Installation Guide](doc/kb_sdk_dependencies.md).
 
-For the adventurous, there is an experimental Dockerized installation that only requires Docker.  It is known to work on at least Mac and Linux, but has not been extensively tested yet.  For details, see [Dockerized Installation](doc/kb_sdk_dockerized_install.md)
-
 #### Installation Only
 
 System Dependencies:
 
 - Mac OS X 10.8+ or Linux. kb-sdk does not run natively in Windows, but see [here](doc/FAQ.md#windows) for more details.
-- Java JRE 7+ http://www.oracle.com/technetwork/java/javase/downloads/index.html
 - (Mac only) Xcode https://developer.apple.com/xcode
 - git https://git-scm.com
 - Docker https://www.docker.com (for local testing)
 
-Get the SDK:
-
-    git clone https://github.com/kbase/kb_sdk
 
-Pull dependencies and configure the SDK:
+Pull the KBase SDK image.
 
-    cd kb_sdk
-    make bin
+    docker pull kbase/kb-sdk
```

Comment:
Is this the right image name/tag?  Maybe @sychan knows

Reply:
Just to clarify.  This isn't the SDK Base image that Steve is working on.  But we probably do want to sync these things up.  We could push up a new tag for this image (e.g. kbase/kb-sdk:2) that would use the new base images.